Gems/PhysX/Code/Source/EditorColliderComponent.cpp

```diff
@@ -1570,6 +1572,22 @@ namespace PhysX
         m_shapeConfiguration.m_cylinder.m_configuration = Utils::CreatePxCookedMeshConfiguration(samplePoints, scale).value();
     }
 
+    AZ::Aabb EditorColliderComponent::GetWorldBounds()
+    {
+        return GetAabb();
```

Comment:
Do we need to have `IsValid()` check for the world bounds like we do in `GetLocalBounds` or the visibility system is OK about dealing with invalid AABBs?

Reply:
The visibility system is fine with invalid AABBs. The reason I put the IsValid check in the other place was just in case anything weird could happen if you inverse transform a null AABB.